src/draive/parameters/model.py

```diff
@@ -31,14 +32,28 @@ def json_schema(
         indent: int | None = None,
     ) -> str:
         if not_missing(cls.__PARAMETERS_SPECIFICATION__):
-            return json.dumps(
+            return json_schema(
                 cls.__PARAMETERS_SPECIFICATION__,
                 indent=indent,
             )
 
         else:
             raise TypeError(f"{cls.__qualname__} can't be represented using json schema")
 
+    @classmethod
+    def simplified_schema(
```

Comment:
That's a bit of a strech for a `@classmethod` as we use it only to access class variables. Is `cls.__PARAMETERS_SPECIFICATION__` really frozen or it's possible it could change in runtime?

Reply:
we are always using it for a given class, Model is a base class so it will eventually change, it is a class method to postpone the model generation, we could cache it but we would still have to check if the model schema is available at all when accessing it